### ISSUE TITLE
Wrong HTTP status

### ISSUE THREAD

**Issue by mariuszsienkiewicz** [mollie/PrestaShop#941]
### Issue

Issue similar to #894.

Please look at this code:

https://github.com/mollie/PrestaShop/blob/f465a4b8294cafbfd9323df3ca395ee173fdc77a/controllers/front/webhook.php#L96-L112

It's not a good idea to set HTTP Status based on Exception error code... example:

```
PHP message: PHP Fatal error:  Uncaught InvalidArgumentException: The HTTP status code "1000002" is not valid. in /<path>/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Response.php:464
Stack trace:
#0 /<path>/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Response.php(207): Symfony\Component\HttpFoundation\Response->setStatusCode(1000002)
#1 /<path>/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/JsonResponse.php(44): Symfony\Component\HttpFoundation\Response->__construct('', 1000002, Array)
#2 /<path>/modules/mollie/src/Infrastructure/Response/JsonResponse.php(28): Symfony\Component\HttpFoundation\JsonResponse->__construct(Array, 1000002, Array)
#3 /<path>/modules/mollie/src/Infrastructure/Response/JsonResponse.php(50): Mollie\Infrastructure\Response\JsonResponse->__construct(Array, 1000002)
#4 /<path>/" while reading response header from upstream, client: <redacted>, server: <redacted>, request: "POST /module/mollie/webhook HTTP/2.0", upstream: "fastcgi://unix:/<redacted>/php-fpm.sock:", host: "<redacted>"
```

### Environment
* PrestaShop version: 1.7.7.1
* Mollie module version: 6.1.1

**Comment by justelis22** [comment]
Hi @mariuszsienkiewicz, thanks for the information!

I have forwarded everything to our development team, they will overview this and I will get back to you once I have more info.

Best Regards,
Invertus Support team.

**Comment by justelis22** [comment]
@mariuszsienkiewicz Hi there,

This was released in version 6.2.2.

Your feedback is highly appreciated!

Best Regards,
Invertus Support team.
